src/sudoers/tokens.rs

```diff
@@ -259,12 +259,13 @@ impl Token for SimpleCommand {
     }
 
     fn accept(c: char) -> bool {
-        !Self::escaped(c) && !c.is_control()
+        // '=' is allowed both escaped and un-escaped
+        ( !Self::escaped(c) || c == '=' ) && !c.is_control()
```

Comment:
```suggestion
        (!Self::escaped(c) && !c.is_control()) || c == '='
```

reads a bit more clear to me.

Reply:
I agree! My intention was to show that `=` was both an escaped character but also accepted, but this reads clearer.